x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/cluster/ClusterStatsMonitoringDocTests.java

```diff
@@ -422,6 +422,7 @@ public void testToXContent() throws IOException {
                 + "      },"
                 + "      \"store\": {"
                 + "        \"size_in_bytes\": 0,"
+                + "        \"local_size_in_bytes\": 0,"
```

Comment:
Since this is mapped as a dynamic=false object, I chose the easy path here, but it is slightly inconsistent with index and indices stats monitoring docs.

Reply:
I don't follow this - what is the mapping you mean?

Reply:
The mapping is in monitoring-es.json here:

https://github.com/elastic/elasticsearch/blob/0b567d60e88f38c2333b97f6861e304d7979479d/x-pack/plugin/core/src/main/resources/monitoring-es.json#L482